lib/uk/ac/ebi/interpro/ProcessOutputGFF3.groovy

```diff
@@ -180,11 +180,9 @@ class ProcessOutputGFF3 {
         def score = null
         if (match.source == "InterPro-N") {
             score = loc.score
```

Comment:
For some of the member db's we include synonyms, for example, for Mobidb-lite we allow `mobidblite`, `mobidb-lite`, `mobidb_lite`, and `cath-funfam` and `funfam` for FunFam. Do we need support all these synonyms, if we do, do we need to do something similar for InterPro-N?

Reply:
> Do we need support all these synonyms

The synonyms are handled in [InterProScan.validateApplications()](https://github.com/ebi-pf-team/interproscan6/blob/121ee535502024db850b301f170569c96e1f6d5e/lib/uk/ac/ebi/interpro/InterProScan.groovy#L270). The value of `memberDb` is the one updated in the `XREFS` process (or hard-coded when creating a new `Match` object for applications that are not member databases such as SignalP). 

> do we need to do something similar for InterPro-N

For the source being `InterPro-N`, it's assigned after creating the `Match` object as well ([link](https://github.com/ebi-pf-team/interproscan6/blob/121ee535502024db850b301f170569c96e1f6d5e/modules/interpro-n/main.nf#L254)). 

Not sure if this answers your question.